ports/icu/portfile.cmake

```
@@ -35,7 +36,10 @@ set(DEBUG_TRIPLET ${TARGET_TRIPLET}-dbg)
 
 if(NOT "${TARGET_TRIPLET}" STREQUAL "${HOST_TRIPLET}")
```

While you're here, change this condition also
```suggestion
if(VCPKG_CROSSCOMPILING)
```